doc/source/user/how-to-verify-bug.rst

```diff
@@ -0,0 +1,153 @@
+.. _how-to-verify-bug:
+
+##############################################################################
+Verifying bugs and bug fixes in NumPy
+##############################################################################
+
+In this how-to you will learn how to:
+
+- Verify the existence of a NumPy bug
+- Verify the fix, if any, made for the bug
+
+While you walk through the verification process, you will learn how to: 
+
+- Set up a Python virtual environment (virtualenv)
+- Install appropriate versions of NumPy, first to see the bug in action, then to verify its fix
+
+`Issue 16354 <https://github.com/numpy/numpy/issues/16354>`_ is used as an example.
+
+This issue was:
+
+.. code-block:: none
+
+    Title:
+    `np.polymul return type is np.float64 or np.complex128 when given an
+    all-zero argument`
+
+    np.polymul returns an object with type np.float64 when one argument is all
+    zero, and both arguments have type np.int64 or np.float32. Something
+    similar happens with all zero np.complex64 giving result type np.complex128.
+    
+    This doesn't happen with non-zero arguments; there the result is as 
+    expected.
+    
+    This bug isn't present in np.convolve.
+    
+    Reproducing code example:
+    >>> import numpy as np
+    >>> np.__version__
+    '1.18.4'
+    >>> a = np.array([1,2,3])
+    >>> z = np.array([0,0,0])
+    >>> np.polymul(a.astype(np.int64), a.astype(np.int64)).dtype
+    dtype('int64')
+    >>> np.polymul(a.astype(np.int64), z.astype(np.int64)).dtype
+    dtype('float64')
+    >>> np.polymul(a.astype(np.float32), z.astype(np.float32)).dtype
+    dtype('float64')
```

Comment:
```suggestion
    dtype('...') # Was float32
```

The issue is that the doctests will run with the corrected version of `numpy` so this will fail (by correctly returning `float32`).